### PR TITLE
Added pre-demo email template

### PR DESCRIPTION
**Pull request by lexnederbragt**

### CODE REVIEW

File: _extras/guide.md [hunk 1]
```diff
@@ -172,6 +172,23 @@ Any episode other than those listed below should make an okay starting point for
 
 ## IX. Email Templates
 
+### Reminder teaching demo
+
+Subject: Software/Data Carpentry teaching demonstration
+
+Hi,
+
+According to http://pad.software-carpentry.org/teaching-demos you have signed up to give an online teaching demo on [date] at [time] (See this link for your local time: [timeanddate.com link]). I will be the instructor training running the session.
```
Comment (ErinBecker) [inline _extras/guide.md]: instructor training --> Instructor Trainer

File: _extras/guide.md [hunk 2]
```diff
@@ -172,6 +172,23 @@ Any episode other than those listed below should make an okay starting point for
...
+
+I wanted to make it very clear that I may give you any segment of the lesson you prepared to teach, so you must be ready to teach any part of your chosen lesson. Some people prepare to teach 5 minutes from a particular section and nothing else, and these often have to reschedule as they seldom are assigned the section they have prepared for. A lesson corresponds to a single line in the lesson tables (https://software-carpentry.org/lessons/ and http://www.datacarpentry.org/lessons/) and a single repository on GitHub. Some lessons have supplementary modules, but you do not need to be prepared to teach the supplementary modules for your teaching demonstration.
+
+For example, if you have chosen The Unix Shell, I may assign you any episode listed at http://swcarpentry.github.io/shell-novice/.
+
+Please visit the etherpad a little while before the demo starts, I will post a link there to another etherpad we will be using for this demo.
```
Comment (ErinBecker) [inline _extras/guide.md]: Please capitalize Etherpad. @gvwilson's request to me many moons ago ;-)

**Comment by ErinBecker** [comment]
Thanks @lexnederbragt - this is very helpful. Just a few minor changes.

**Comment by lexnederbragt** [comment]
Thanks. All set, methinks!

**Comment by ErinBecker** [comment]
Awesome, thanks @lexnederbragt!

**Pull request merged**